doc/source/contributing.rst

```diff
@@ -175,11 +175,11 @@ nor any existing python installation.
 
 To work in this environment, Windows users should ``activate`` it as follows::
```

Comment:
In theory it's the same for windows, mac and linux now SO we could merge these together if you would like to!

Reply:
Sure, there is no reason anymore to keep it separately if it contains the same instructions.

Reply:
ok!